### ISSUE TITLE
[bug] Docs dropdown only shows `v1.2.0` and `v0.7.4`

### ISSUE THREAD

**Issue by EndangeredMassa** [tailwindlabs/tailwindcss.com#351]
The docs dropdown only show `v1.2.0` and `v0.7.4`. As someone currently using `v1.1.0`, I want to be able to use the docs without wondering if a given feature is available to me or not.

I think this can be solved in one of two ways:

1. Show all `major.minor` versions in the dropdown
1. Add a rather visible notice to each feature that describes what version it was introduced

---

<img width="483" alt="Screen Shot 2020-02-06 at 4 22 30 PM" src="https://user-images.githubusercontent.com/41545/73983807-e50cc800-48fc-11ea-83d5-8339f9c933ec.png">

**Comment by EndangeredMassa** [comment]
It's not only name clashes. There can always be bugs and therefore there is always at least some risk in updating.

By not making the specific docs version available, there is now pressure on my team to upgrade this dependency. Pressure and risk don't mix well.

It's also important to consider that dependencies can be transitive. Users may be depending on different library that depends on tailwind, perhaps something that adds it to your preferred framework. Now there would be pressure on the third party to upgrade so that the end user could upgrade.

Yes, as you mentioned, the values for both pressure and risk are relatively small here. It's not a _huge_ deal. It is something that I think will benefit the project, especially as it grows.

**Comment by simonswiss** [comment]
Those are valid points!

I don't think realistically we'll be able to provide docs for each version of Tailwind. We've tried to have clear version badges for features that were added later, to add some context on the features available in different versions.

We'll keep this suggestion in mind though - I have captured it internally. If there is growing demand for it that warrants putting other projects on pause, we'll consider it! 👍
